.github/workflows/test-executor-linux-arm64.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Test linux/arm64 executor"
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04-16cpu-arm64
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # GitHub's arm64 runners do not have the same pre-installed software as
+      # the amd64 runners so we have to install a few things here.
+
+      - name: Install bazelisk
+        run: |
+          curl -L --output /tmp/bazelisk https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-arm64
+          chmod +x /tmp/bazelisk
+          sudo mv /tmp/bazelisk /usr/bin/bazelisk
+          sudo ln -s /usr/bin/bazelisk /usr/bin/bazel
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc g++ podman
```

Comment:
Could we run these in an ARM64 container? 🤔 
https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

It could simplify this setup by quite a bit.

Reply:
Container jobs aren't supported in the arm runners beta yet - idk if it's mentioned in the public docs but they mentioned that in their email.